scvi/_settings.py

```diff
@@ -3,8 +3,8 @@
 from pathlib import Path
 from typing import Literal, Union
 
-import pytorch_lightning as pl
 import torch
+from lightning_lite import seed_everything
```

Comment:
how does this get installed?

Reply:
It's installed along with `pytorch-lightning`